### DISCUSSION TITLE
Feature request: Allow option to skip verification when decrypting

### DISCUSSION THREAD

**Discussion by chadmyers** [openpgpjs/openpgpjs#1783] [Ideas]
Similar to the gpg [`--skip-verify` option](https://www.gnupg.org/gph/en/manual/r1564.html).

If I receive an encrypted PGP file and the sender has added an optional signature, but I don't care about the signature verification, I should be able to decrypt the file and skip the verification step.

**Comment by larabr** [comment]
Hi, it's already the case the verification failure does not cause decryption failure, unless the option `expectSigned: true` is given; you can refer to the [README](https://github.com/openpgpjs/openpgpjs?tab=readme-ov-file#encrypt-and-decrypt-string-data-with-pgp-keys) for examples on how to use the `decrypt` function.

**Reply by chadmyers** [reply]
Thank you!  I did read the docs and the code thoroughly but it wasn't clear to me whether or not it was required. I was getting an error but it turns out the error was related to something else. Thanks!
